spec/models/cartitem_spec.rb

```diff
@@ -5,9 +5,15 @@
   let(:r) { create_restaurant("Boston Beer Garden") }
   let(:cart) { FactoryGirl.create(:cart, user: user, restaurant: r) }
   let(:item) { FactoryGirl.create(:item, restaurantdetail: r.restaurantdetail) }
+  let(:longnameitem) { FactoryGirl.create(:item, :longname,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.